### DISCUSSION TITLE
Styling and Themes?

### DISCUSSION THREAD

**Discussion by pietz** [N00nDay/stwui#57] [General]
Are there any features planned that allow me to easily modify the current styling of elements? Other frameworks do this through customizable themes. What's your recommended approach for changing default colors and looks as of today? Also the hover animation is a little much for my taste.

Thanks

**Comment by N00nDay** [comment]
This is definitely on the roadmap but there is no overarching theming at this time. Light and Dark modes work out of the box (allow I just realized it is not documented). In the future, there will be a theme generator to help create themes and a way to add as many themes as you desire. At this time I am still focused on functionality as there is still a lot of work to be done in that area.

**Reply by N00nDay** [reply]
I agree with the Hover animation; more importantly, it is not customizable across the library. You can opt in/out for some components but not others and have no control over what it looks like. This will be swapped out with a tailwind `hover:` effect that developers can easily override.

**Comment by N00nDay** [comment]
@pietz Themes have landed for STWUI! check them in the latest release! The announcement can be found here: https://github.com/N00nDay/stwui/discussions/101.
